### PR TITLE
Cost split

### PR DESCRIPTION
**Pull request by Nisheanthan**
-added split by percentage option 
-handled  relevant errors to display

### CODE REVIEW

File: costSplit/index.html [hunk 1]
```diff
@@ -7,31 +7,97 @@
...
       <h2>Cost Spliter</h2>
-      <input type="number" id="amt" />
-      <input type="number" id="splitCost" />
+
+      <div id="general-split">
+          <label for="people">Number of People: </label>
+          <input type='number' id="people"/>
+      </div>
+      
+      <div id="percentage-split" style="display: none">
```
Comment (Neha) [inline costSplit/index.html]: Move the inline css from DOM to style file

File: costSplit/index.html [hunk 2]
```diff
@@ -7,31 +7,97 @@
...
+        <label for="first_person"> Name: </label>
+          <input type="text" id="first_person"/>
+        <label>percentage: </label>
+          <input type='number' id="first_percentage"/>% <br/>
```
Comment (Neha) [inline costSplit/index.html]: Either has the id namespace _fname-lname_ or _fname_lname._

File: costSplit/index.html [hunk 3]
```diff
@@ -7,31 +7,97 @@
...
+        <label for="second_person"> Name: </label>
+          <input type="text" id="second_person"/>
+        <label>percentage: </label>
+          <input type='number' id="second_percentage"/>%
+      </div>
+
+      <br/>
```
Comment (Neha) [inline costSplit/index.html]: remove <br/> and manage the styling by CSS only.

**Review by Neha — CHANGES_REQUESTED** [review]
Hello,

Please look into the comments.

Thanks

**Comment by Nisheanthan** [comment]
done. Thanks for spending time to make my mistakes, I could learn many thing while involving in this project.

**Comment by Nisheanthan** [comment]
thanks a lot to point out those mistakes. Nice experience working on this. Thank you

**Comment by Neha** [comment]
no response. Hence, closing it

**Pull request closed without merging**